src/join.rs

```diff
@@ -163,6 +168,7 @@ pub trait Join {
 /// The purpose of the `ParJoin` trait is to provide a way
 /// to access multiple storages in parallel at the same time with
 /// the merged bit set.
+#[cfg(feature = "parallel")]
```

Comment:
Meh, this isn't really ideal. One should be able to compile just the same code for wasm, but this disallows `ParJoin`. Instead we'd need a fallback for it. But I'm not sure if we can use rayon at all on wasm :(

Reply:
It seems that `rayon` should build with wasm (rayon-rs/rayon#503), but as wasm doesn't support threads it just panics when using it.

Reply:
Heh, what's the point of being able to compile something only for it to blow up later? I thought this behaviour of wasm32-unknown-unknown target is something that's just done so that it can be shipped earlier. Anyhow, I can change it if you wish

Reply:
Unless the fallback would be to define ParJoin as Join?

Reply:
I wasn't implying we should use it, but just providing info.